### DISCUSSION TITLE
5QI implementation

### DISCUSSION THREAD

**Discussion by hejazifar** [srsran/srsRAN_Project#153] [General Help] [answered]
Dear community

I am currently in search of a simulator that incorporates the 5QI (5G Quality of Service Identifier) for analyzing various network parameters, such as latency, bandwidth, priority, and more. Specifically, I would like to be able to compare and observe the differences when selecting different 5QI values, such as 3 and 80.

With that in mind, I wanted to inquire if srsRAN_Project has implemented this particular feature.

Thank in advance.
Mehdi

**Comment by andrepuschmann** [comment, marked as answer]
Hey @hejazifar, we have configurable 5QI settings for the CU/DU - see https://github.com/srsran/srsRAN_Project/blob/main/configs/qos.yml You can configure the values you're interested in and run your desired experiments. Make sure to pass the (modified) config file with the `-c` option whenever you start the gNB.
